content/docs/analytics/Overview.md

```diff
@@ -98,6 +101,28 @@ Search Latency view lets you visualize your query distribution and show you slow
 
 ![Search Latency](https://miro.medium.com/max/2062/1*Tq_4WZAQFeqTNTHArmmS0g.png)
 
+#### User Metrics
+
+![alt User Metrics](https://i.imgur.com/PIJUb9U.png)
+
+**Total Users**
+helps you to find the total number of distinct users who have performed at least one search in your application.
+
+**Total Sessions**
+represents the total number of user sessions, a user session can be defined as a group of `search` interactions one user takes in the given time frame on your application.
+
+**Session Duration(Avg.)**
```

Comment:
@bietkul I thought more on this. We should call it `Avg Session Duration` and also label it this way in the dashboard.